src/findings/utils.py

```diff
@@ -84,7 +84,7 @@ def __init__(
 
     def toDict(self):
         return {
-            "category": self.category.value,
+            "category": self.category.name,
```

Comment:
`.value` gives us the number which is not that useful, `.name` returns the name of the category (e.g. enum_removal)